vendor/vendor.json

```diff
@@ -922,10 +928,10 @@
 			"revisionTime": "2017-06-05T02:18:34Z"
 		},
 		{
-			"checksumSHA1": "QHuUJtX9TiA32UXIhsQSe6x4qVA=",
```

Comment:
Had to update it to get the native Go's for getting the health of a vault, see sys_health.go.